src/pykeen/typing.py

```diff
@@ -148,3 +158,17 @@ class GaussianDistribution(NamedTuple):
 EA_SIDE_LEFT: EASide = "left"
 EA_SIDE_RIGHT: EASide = "right"
 EA_SIDES: Tuple[EASide, EASide] = (EA_SIDE_LEFT, EA_SIDE_RIGHT)
+
+# temporal typing
+TimestampMapping = Mapping[str, int]
+LabeledQuadruples = np.ndarray
+MappedQuadruples = torch.LongTensor
+
+TimestampRepresentation = TypeVar("TimestampRepresentation", bound=OneOrSequence[torch.FloatTensor])
+
+#: the prediction target
+Target = Literal["head", "relation", "tail", "timestamp"]
```

Comment:
Re-definition of existing constants.

I think you need to establish a new constant, e.g., `TemporalTarget` to avoid name collision, since not all code paths support passing timestamps for now.